DOCS.md

```diff
@@ -265,12 +265,23 @@ Possible values:
   * between `\[` and `\]` delimiters - example: `\[ a=-\sqrt{2} \]`
   * between `\(` and `\)` delimiters - example: `\( b=2 \)`
 
-#### **org_indent_mode**
-*type*: `string`<br />
-*default value*: `indent`<br />
+#### **org_startup_indented**
+
+*type*: `boolean`<br />
+*default value*: `false`<br />
 Possible values:
-* `indent` - Use default indentation that follows headlines/checkboxes/previous line indent
-* `noindent` - Disable indentation. All lines start from 1st column
+* `true` - Enables indent mode on startup. Lines that follow headlines are indented to that headline level.
+* `false` - Disables indent mode on startup. All lines start from the first column.
+
+#### **org_adapt_indentation**
+
+*type*: `boolean`<br />
+*default value*: `true`<br />
+Possible values:
+* `true` - Converts indents to *Virtual* indents. The indents are only visual, they are not saved  to the file.
```

Comment:
Values are reverted here. When it's `true`, it should use real indents.
In Emacs, enabling `org-indent-mode` automatically sets this value to `false` ([docs](https://orgmode.org/manual/Org-Indent-Mode.html#:~:text=By%20default%2C%20Org%20Indent%20mode%20turns%20off%20org%2Dadapt%2Dindentation)), but we will not actually modify any of the settings, just consider a priority.

If `org_startup_indented = true`, always use virtual indents.
If `org_startup_indented = false`, follow `org_adapt_indentation` according to the describe logic above.

Reply:
Ah, I see... https://orgmode.org/manual/Hard-indentation.html. I'm a bit off, I already have a lot of this in my Emacs config that I'm too used to 🙂. 

I'll correct this momentarily.

Reply:
@kristijanhusak 

Done. I made `adapt_indentation` *very* similar to Orgmode.

When `org_startup_indented` is disabled and `org_adapt_indentation` is enabled you only get hard indentation, just like on `master` currently.

When `org_startup_indented` is enabled and `org_adapt_indentation` is disabled you get only Virtual Indentation.

When `org_startup_indented` is true and `org_adapt_indentation` is enabled you get *both* Virtual indentation *and* hard indents like Emacs.